### PR TITLE
Remove `ValidateTokens()` from `IAntiforgery`

### PR DESCRIPTION
**Pull request by dougbu**
- `IAntiforgery` does not expose a way to get an invalid `AntiforgeryTokenSet`

### CODE REVIEW

**Comment by dougbu** [comment]
/cc @rynowak 
FYI @Eilon teensy removal from public surface

File: src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgery.cs [hunk 1]
```diff
@@ -135,8 +135,8 @@ public AntiforgeryTokenSet GetTokens(HttpContext httpContext)
             ValidateTokens(httpContext, tokens);
         }
 
-        /// <inheritdoc />
-        public void ValidateTokens(HttpContext httpContext, AntiforgeryTokenSet antiforgeryTokenSet)
+        // Internal for testing
+        internal void ValidateTokens(HttpContext httpContext, AntiforgeryTokenSet antiforgeryTokenSet)
```
Comment (rynowak) [inline src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgery.cs]: IMO either just make it private and remove these tests. The tests are there basically for argument validation IIRC and there's a fair bit of duplication.

**Comment by dougbu** [comment]
:up::date:

File: src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgery.cs [hunk 2]
```diff
@@ -135,8 +135,8 @@ public AntiforgeryTokenSet GetTokens(HttpContext httpContext)
             ValidateTokens(httpContext, tokens);
         }
 
-        /// <inheritdoc />
-        public void ValidateTokens(HttpContext httpContext, AntiforgeryTokenSet antiforgeryTokenSet)
+        // Internal for testing
```
Comment (rynowak) [inline src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgery.cs]: Nuke comment

**Comment by rynowak** [comment]
:shipit:

**Comment by dougbu** [comment]
08cf13b

**Pull request closed without merging**